scripts/fast-reboot

```diff
@@ -723,9 +703,11 @@ for service in ${SERVICES_TO_STOP}; do
             if [[ "x$sonic_asic_type" == x"mellanox" ]]; then
                 check_issu_bank_file
             fi
+        fi
 
-            # Warm reboot: dump state to host disk
-            if [[ "$REBOOT_TYPE" = "fastfast-reboot" ]]; then
+        if [[ "$REBOOT_TYPE" = "warm-reboot" || "$REBOOT_TYPE" = "fastfast-reboot" || "$REBOOT_TYPE" = "fast-reboot" ]]; then
+            # Advanced reboot: dump state to host disk
+            if [[ "$REBOOT_TYPE" = "fastfast-reboot" || "$REBOOT_TYPE" = "fast-reboot" ]]; then
```

Comment:
Performance wise I agree that is pricey (especially when each condition check is actually checking multiple conditions), however I think Shlomi's intent was differentiate  warmboot from fastboot and fastfastboot, i.e. in warmboot won't flush asic/counters/flexcounters db. What do you suggest in this case?

Reply:
You have this logic:
```
if warm or fastfast or fast:
   if fastfast or fast
      flush DBs
   backup DBs
```

While it can be written as:

```
if fast or fastfast:
  flush DBs
backup DBs
```

Reply:
Modified